agents/ebner_agent.py

```diff
@@ -40,9 +41,11 @@ def _make_modulatory_population(self, name, cell_num, source=None, syn_per_cell=
         pop = EbnerModulatoryPopulation(name)
         pop.create(cell_num)
 
+        # info draw netcon_weights from N(WEIGHT, epsilon)
+        # todo what should be the epsilon value? WEIGHT is small (=0.0035)
         syns = pop.connect(source=source, syn_num_per_source=syn_per_cell,
-                           delay=1, netcon_weight=WEIGHT, ach_weight=1, da_weight=1, rule='all',
-                           ACh_tau=50, Da_tau=50)
+                           delay=1, netcon_weight=np.max((0, WEIGHT + np.random.normal(loc=0.0, scale=EPSILON))),
+                           ach_weight=1, da_weight=1, rule='all', ACh_tau=50, Da_tau=50)
```

Comment:
I have doubt about netcon_weight to make it random - let's talk on the call

Reply:
As of now, reject these modifications. It needs deeper tests.